packages/split-button/README.md

```diff
@@ -0,0 +1,57 @@
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/split-button
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/split-button
+```
+
+## Example
+
+```js
+
+import { SplitButton } from `@leafygreen-ui/split-button`;
+import { MenuItem } from '@leafygreen-ui/menu';
+
+<SplitButton
+  label="label"
+  menuItems={
+    <>
+      <MenuItem>Menu Item</MenuItem>
+      <MenuItem disabled>Disabled Menu Item</MenuItem>
+      <MenuItem description="I am also a description">
+        Menu Item With Description
+      </MenuItem>
+    </>
+  }
+/>
+```
+
+## Properties
+
+| Prop              | Type                                                | Description                                                                                                                                                                                                                                      | Default     |
+| ----------------- | --------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ----------- |
+| `label`           | `string`                                            | The text that will appear inside of the primary button.                                                                                                                                                                                          |             |
+| `darkMode`        | `boolean`                                           | Renders the component with dark mode styles.                                                                                                                                                                                                     | `false`     |
+| `variant`         | `'default'` \| `'primary'` \| `'danger'`            | Sets the variant for both Buttons.                                                                                                                                                                                                               | `'default'` |
+| `size`            | `'xsmall'` \| `'small'` \| `'default'` \| `'large'` | Sets the size for both buttons.                                                                                                                                                                                                                  | `'default'` |
+| `align`           | `'top'` \| `'bottom'`                               | Determines the alignment of the menu relative to the component wrapper.                                                                                                                                                                          | `'bottom'`  |
```

Comment:
why are we exposing only a subset of the available alignments/justifications

Reply:
I'm going to double-check with Rob; I was going off of the specs in Figma which only showed alignment `top` and `bottom` and for the justification `start` and `end`

Reply:
this is correct, we don't want consumers to select middle, center, etc...